application/ledger_query_service/src/ethereum/queries.rs

```diff
@@ -408,63 +493,200 @@ mod tests {
 
     #[test]
     fn given_a_transaction_receipt_should_not_match_empty_query() {
```

Comment:
:x: I think an empty query *should* match any transaction receipt.

Reply:
done